k8s/apply.sh.tpl

```diff
@@ -16,9 +16,14 @@
 set -euo pipefail
 
 function guess_runfiles() {
-    pushd ${BASH_SOURCE[0]}.runfiles > /dev/null 2>&1
-    pwd
-    popd > /dev/null 2>&1
+    if [ -d ${BASH_SOURCE[0]}.runfiles ]; then
+        # Runfiles are adjacent to the current script.
+        echo "$( cd ${BASH_SOURCE[0]}.runfiles && pwd )"
```

Comment:
This could also be written as:
```sh
(cd ${BASH_SOURCE[0]}.runfiles && pwd)
```

But I'd rather not deviate from what rules_docker does.